extra/cache-extra/src/CacheRuntime.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Extra\Cache;
+
+use Symfony\Contracts\Cache\CacheInterface;
+
+class CacheRuntime
+{
+    private $cache;
+
+    public function __construct(CacheInterface $cache)
```

Comment:
Don't we have a TagAwareCacheInterface?

Reply:
When using the default Symfony configuration from the extra bundle, yes. But we accept `CacheInterface` as well as using tags is optional (see the Node, we catch the exception and ignore it if we don't have tag support). Is there a better way?

Reply:
Then throw an exception when using tags with a non compatible item/pool?

Reply:
Not sure this is a good idea as if a bundle use some caches with tags, I still want people to be able to use a "standard" CacheInterface.